app/src/main/res/drawable/corner_radius_title.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android" >
+    <item >
+        <shape android:shape="rectangle"  >
+            <corners android:radius="50dp" />
+            <stroke android:width="3dp" android:color="@color/white"/>
```

Comment:
hardcoded dp

Reply:
How do you get a standard size? I don't know how to put it.